lib/routes/withdraw/reverse_swap/confirmation_page/widgets/reverse_swap_button.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/bloc/withdraw/withdraw_funds_bloc.dart';
+import 'package:c_breez/utils/exceptions.dart';
+import 'package:c_breez/widgets/error_dialog.dart';
+import 'package:c_breez/widgets/loader.dart';
+import 'package:c_breez/widgets/single_button_bottom_bar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class ReverseSwapButton extends StatelessWidget {
+  final int amountSat;
+  final String onchainRecipientAddress;
+  final String feesHash;
+  final int satPerVbyte;
+
+  const ReverseSwapButton({
+    Key? key,
+    required this.amountSat,
+    required this.onchainRecipientAddress,
+    required this.feesHash,
+    required this.satPerVbyte,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final texts = context.texts();
+
+    return SingleButtonBottomBar(
+      text: texts.sweep_all_coins_action_confirm,
+      onPressed: () => _sendOnchain(context),
+    );
+  }
+
+  Future _sendOnchain(BuildContext context) async {
+    final texts = context.texts();
+    final withdrawFundsBloc = context.read<WithdrawFundsBloc>();
```

Comment:
We need to totally separate the reverse swap from the withdraw funds, also here.
These are different things using the same name was part of the lately confusion regarding the implementation.